packages/frontend/src/lib/Navigation.svelte

```diff
@@ -69,6 +70,16 @@ onDestroy(() => {
     <SettingsNavItem icon={faRocket} title="Services" selected={meta.url === '/services'} href="/services" />
     <SettingsNavItem icon={faMessage} title="Playgrounds" selected={meta.url === '/playgrounds'} href="/playgrounds" />
 
+    <!-- Settings -->
+    <div class="pl-3 mt-2 ml-[4px]">
+      <span class="text-[color:var(--pd-secondary-nav-header-text)]">Server Information</span>
+    </div>
+    <SettingsNavItem
+      icon={faGear}
+      title="Local Server"
+      selected={meta.url === '/ai_lab_service'}
+      href="/ai_lab_service" />
```

Comment:
name of the url/title should match the parent

like : `/server-information/service`